nuxeo-features/nuxeo-platform-forms-layout-export/pom.xml

```diff
@@ -35,22 +35,6 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
     </dependency>
-    <!-- tests dependencies -->
-    <dependency>
-      <groupId>org.skyscreamer</groupId>
-      <artifactId>jsonassert</artifactId>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.nuxeo.ecm.platform</groupId>
-      <artifactId>nuxeo-platform-forms-layout-client</artifactId>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.nuxeo.ecm.platform</groupId>
-      <artifactId>nuxeo-features-test</artifactId>
-      <scope>test</scope>
-    </dependency>
   </dependencies>
```

Comment:
can you please confirm layout-export stays independent of JSF please?

Reply:
Yes `nuxeo-platform-forms-layout-export` stays in `nuxeo-features`. Only the tests are moved to `nuxeo-platform-forms-layout-client` because they require setup from this module (which is JSF-specific).